### PR TITLE
Added Python 3.12 to test matrix and classifiers

### PR DESCRIPTION
**Pull request by foarsitter**

### CODE REVIEW

**Comment by ThiefMaster** [comment]
You may want to consider updating cibuildwheel as well - I think at least [v2.16.2](https://github.com/pypa/cibuildwheel/releases/tag/v2.16.2) is needed to build cp312 wheels. IMHO it's a good idea to include this with your PR since you also declare Python 3.12 support, so it'd a bit strange if it'd still do a release without the proper wheels.

**Comment by foarsitter** [comment]
Makes indeed no sense @ThiefMaster, thanks for pointing out! I updated the pull-request.

**Comment by ThiefMaster** [comment]
great, now we just need a maintainer to merge this stuff and make a release :)

👀 @chayim @badboy @pietern

**Comment by badboy** [comment]
[I don't maintain hiredis-py anymore](https://fnordig.de/2022/11/06/state-of-maintenance-hiredis-py/)

File: .github/workflows/integration.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ jobs:
      strategy:
        max-parallel: 15
        matrix:
-         python-version: ['3.7', '3.8', '3.9', '3.10', '3.11', 'pypy-3.7', 'pypy-3.8']
+         python-version: ['3.7', '3.8', '3.9', '3.10', '3.11', '3.12', 'pypy-3.7', 'pypy-3.8']
```
Comment (chayim) [inline .github/workflows/integration.yaml:28]: Let's add pypy-3.9 while here, WDYT?

**Comment by chayim** [comment]
Just the one comment @foarsitter. I'm happy to push to it, if you prefer.

File: .github/workflows/integration.yaml (hunk 1 above)
Comment (foarsitter) [reply .github/workflows/integration.yaml:28]: Feel free to commit, I have no objection.

**Comment by foarsitter** [comment]
Thanks @chayim!

**Pull request merged**